tests/manage/pv_services/pvc_resize/test_resource_deletion_during_pvc_expansion.py

```diff
@@ -0,0 +1,186 @@
+import logging
+import pytest
+from concurrent.futures import ThreadPoolExecutor
+
+from ocs_ci.ocs import constants
+from ocs_ci.framework.testlib import (
+    skipif_ocs_version, ManageTest, tier4, tier4b, ignore_leftover_label
+)
+from tests import disruption_helpers
+
+log = logging.getLogger(__name__)
+
+
+@tier4
+@tier4b
+@skipif_ocs_version('<4.5')
+@ignore_leftover_label(constants.drain_canary_pod_label)
+@pytest.mark.parametrize(
+    argnames='resource_to_delete',
+    argvalues=[
+        pytest.param(
+            'mgr', marks=pytest.mark.polarion_id('OCS-2224')
+        ),
+        pytest.param(
+            'osd', marks=pytest.mark.polarion_id('OCS-2225')
+        ),
+        pytest.param(
+            'rbdplugin', marks=pytest.mark.polarion_id('OCS-2226')
+        ),
+        pytest.param(
+            'cephfsplugin', marks=pytest.mark.polarion_id('OCS-2227')
+        ),
+        pytest.param(
+            'rbdplugin_provisioner', marks=pytest.mark.polarion_id('OCS-2228')
+        ),
+        pytest.param(
+            'cephfsplugin_provisioner',
+            marks=pytest.mark.polarion_id('OCS-2229')
+        )
+    ]
+)
+class TestResourceDeletionDuringPvcExpansion(ManageTest):
+    """
+    Tests to verify PVC expansion will be success even if rook-ceph, csi pods
+    are re-spun during the expansion
+
+    """
+    @pytest.fixture(autouse=True)
+    def setup(
```

Comment:
quite similar to other setup fixtures under pv_services test files. 
How about consolidating them into a function and call them from the test classes?

Reply:
Added fixture for creating pvcs and pods.
Test cases in tests/manage/pv_services/pvc_resize/ can be modified to use this fixture once this is merged.

Reply:
thanks!